src/routes.ts

```diff
@@ -4,5 +4,12 @@ export const Routes = {
   taxonsPath: () => 'taxons',
   taxonPath: (id: string) => `taxon/${id}`,
   countriesPath: () => 'countries',
-  countryPath: (iso: string) => `countries/${iso}`
+  countryPath: (iso: string) => `countries/${iso}`,
```

Comment:
Will be great, if routes will be in alphabetical order within one part, for example: `taxons`, `cart`.